### ISSUE TITLE
traffic sign detection

### ISSUE THREAD

**Issue by a-drop-in-the-ocean456** [HouraisanMokou/traffic_detect#1]
hello ,i also work on traffic sign detection , i found tt100k pictures are too large ,now i want to crop into smaller pieces, but face with difficult,do you konw how to do that?

**Comment by HouraisanMokou** [comment]
Actually I am only a beginner of traffic sign detection, and my suggestion may not help... If you are also using **Pytorch**, you may try **torchvision.transforms** where several transform operation such as  random crop and resize is implemented. Detailed instruction can be found [here](https://pytorch.org/vision/stable/transforms.html#torchvision-transforms). Another good way to crop the image is through **OpenCV** (However **OpenCV** do not support Python 3.8 yet). If you just want to resize the image, PIL can also do that
The only problem is that cropping the image while training may cost longer time. Thus, croping the image while preprocessing, saving them as new files and finally loading image from new files while training may be a better way.

**Comment by a-drop-in-the-ocean456** [comment]
Yes,before train cut into pieces,so that cost less CUDA memory,but i can not found code to do that , some paper do that but do not give code.

**Comment by a-drop-in-the-ocean456** [comment]
and i am also a beginner at traffic sign detection.
